scripts/android.gradle

```diff
@@ -17,4 +17,9 @@ android {
     kotlinOptions {
         jvmTarget = JavaVersion.VERSION_11.toString()
     }
+
+    tasks.withType(JavaCompile).configureEach {
+        this.targetCompatibility = JavaVersion.VERSION_11.toString()
+        this.sourceCompatibility = JavaVersion.VERSION_11.toString()
+    }
```

Comment:
Since AGP 8.X forces Java 17 to compile a project, each package will able to be compiled with Java 11 in client projects by adding this.